account_financial_report_webkit/report/common_partner_balance_reports.py

```diff
@@ -259,7 +261,7 @@ def compute_partner_balance_data(self, data, filter_report_type=None):
         # get details for each accounts, total of debit / credit / balance
         accounts_by_ids = self._get_account_details(
             account_ids, target_move, fiscalyear, main_filter, start, stop,
-            initial_balance_mode)
+            initial_balance_mode, context=lang_ctx)
```

Comment:
@luc-demeyer I need this (https://github.com/luc-demeyer/account-financial-reporting/pull/1) for a customer project. Can you have a look please ? Thanks